app/screens/settings/setting_container.tsx

```diff
@@ -17,7 +17,7 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
             backgroundColor: theme.centerChannelBg,
         },
         contentContainerStyle: {
-            marginTop: 8,
+            marginTop: 24,
```

Comment:
I don't think this needed to change.....adding this has added extra space at the top that we removed in the settings UI polish PR. `marginTop: 8` should be correct